packages/polyfills-loader/polyfills-loader.js

```diff
@@ -20,8 +20,8 @@ function needsTemplatePolyfill() {
   template.content.appendChild(template2);
   const clone = template.cloneNode(true);
   return (
-    clone.content.childNodes.length !== 0 &&
-    clone.content.firstChild.content.childNodes.length !== 0
+    clone.content.childNodes.length === 0 ||
```

Comment:
let's rebase => that's already on master

Reply:
I was going to squad this MR, that would achieve the same, right?

Reply:
I rebased and force pushed